src/geometry/manifold/diffeomorph.lean

```diff
@@ -0,0 +1,81 @@
+/-
+Copyright © 2020 Nicolò Cavalleri. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Nicolò Cavalleri.
+-/
+
+import geometry.manifold.times_cont_mdiff_map
+
+/-!
+# Diffeomorphisms
+This file implements diffeomorphisms.
```

Comment:
Can you expand the docstring, mentioning the definition, the notations, and comparing to structomorphisms (explaining that the notion here is more flexible as you don't require `H = H'`)